src/Take.Blip.Builder/FlowManager.cs

```diff
@@ -92,6 +96,21 @@ public async Task ProcessInputAsync(Message message, Flow flow, CancellationToke
                 throw new ArgumentNullException(nameof(flow));
             }
 
+            if (message.Content is InputExpirationTimeDocument)
```

Comment:
You can use the new Pattern Matching syntax to simplify this condition like this:

``` csharp
if (message.Content is InputExpirationTimeDocument inputTimeExpiration)
{
   // Now inputTimeExpiration is in the scope, you won't need the casting anymore
    if (string.IsNullOrEmpty(inputTimeExpiration?.Identity?.ToString()) 
    {
        ....
```